includes/Checker/Preparations/Demo_Posts_Creation_Preparation.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Class WordPress\Plugin_Check\Checker\Preparations\Demo_Posts_Creation_Preparation
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\Checker\Preparations;
+
+use WordPress\Plugin_Check\Checker\Preparation;
+use Exception;
+
+/**
+ * Class to create demo posts to be used by Checks.
+ *
+ * @since n.e.x.t
+ */
+class Demo_Posts_Creation_Preparation implements Preparation {
+
+	/**
+	 * An array of posts data to create.
+	 *
+	 * @since n.e.x.t
+	 * @var arrat
```

Comment:
```suggestion
	 * @var array
```

Reply:
@mukeshpanchal27 I've addressed the typo, please can review and approve, thanks!